ragna/deploy/_ui/central_view.py

```diff
@@ -447,34 +447,29 @@ def chat_interface(self):
                     ),
                 )
             ],
+            card_params=dict(
+                stylesheets=ui.stylesheets(
```

Comment:
This is not going to work. See holoviz/panel#6240. This was "promised" in https://github.com/Quansight/ragna/pull/261#discussion_r1434489553 although the comment might also refer to making `card_params` usable in the first place. cc @ahuang11

Reply:
Fixed in panel 1.3.7rc3